packages/web-components/src/divider/divider.options.ts

```diff
@@ -0,0 +1,55 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * Align content within divider
+ * @public
+ */
+export const AlignContent = {
+  /**
+   * Default. Center align content.
+   */
+  center: 'center',
+  /**
+   * Align content at flex start
+   */
+  start: 'start',
+  /**
+   * Align content at flex end
+   */
+  end: 'end',
+} as const;
+
+/**
+ * The types for AlignContent
+ * @public
+ */
+export type AlignContent = ValuesOf<typeof AlignContent>;
+
+/**
+ * Appearance is divider color
+ * @public
+ */
+export const Appearance = {
+  /**
+   * Strong corresponds to design token alias.
+   */
+  strong: 'strong',
+  /**
+   * Brand corresponds to design token alias
+   */
+  brand: 'brand',
+  /**
+   * Subtle corresponds to design token alias
+   */
+  subtle: 'subtle',
+  /**
+   * Default corresponds to design token alias
+   */
+  default: 'default',
+} as const;
+
+/**
+ * The types for Appearance
+ * @public
+ */
+export type Appearance = ValuesOf<typeof Appearance>;
```

Comment:
I think we likely will want to import and re-export the Divider const and typing for `DividerRole` from the Foundation package so it's available for use in the `.options.ts` file.

Reply:
I can import `DividerRole`. I can export type DividerRole.  What should the const be? FastDividerRole?

Reply:
I think you can just import at the top of the file and export at the bottom as below:

```
import { DividerRole } from "@microsoft/fast-foundation";

export { DividerRole };
```